Beta Estimation Project Notes.md

```diff
@@ -0,0 +1,13 @@
+## Beta Estimation Project Notes
```

Comment:
Let's move this into the notebooks folder

Reply:
This has been done now,